packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig

```diff
@@ -3,10 +3,10 @@
 {% set link = actionStep.link %}
 
 <section
-  class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
+  class="ma__action-step {{ accordion ? ' js-accordion ma__action-step--accordion ' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">

```

Comment:
```suggestion
  class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">

```

I don't think these spaces are necessary

Reply:
I see the issue here https://mayflower.digital.mass.gov/patternlab/b/develop/?p=templates-location
Withous speaces it was just concating values without spaces

Reply:
really? Are you referring to the page alerts accordions? I'm not seeing spacing issues 
<img width="1498" alt="Screen Shot 2022-09-13 at 9 52 38 AM" src="https://user-images.githubusercontent.com/5789411/189919680-7081740c-88ed-4845-8fb6-68afc4b012cc.png">
The spaces outside the double brackets should be sufficient. 

Also, the molecule seems okay too https://mayflower.digital.mass.gov/patternlab/b/develop/?p=templates-location